Proxy/Configuration/index.ts

```diff
@@ -1,12 +1,19 @@
 import { Request as ConfigurationRequest } from "./Request"
 import { Response as ConfigurationResponse } from "./Response"
 
-export interface Configuration {
+export type Configuration = NormalConfiguration | ReverseConfiguration
+
+interface NormalConfiguration {
 	id: string
 	request: ConfigurationRequest
 	response: ConfigurationResponse.Json
 }
 
+interface ReverseConfiguration {
+	id: string
+	request: ConfigurationRequest
+}
+
```

Comment:
Questions:

1. Could we don't just use one format and make some of the properties optional?
2. If not, I suggest naming it `ForwardConfiguration` and `ReverseConfiguration`.